vendor/github.com/kubernetes-csi/csi-proxy/client/groups/iscsi/v1alpha2/client_generated.go

```diff
@@ -1,98 +0,0 @@
-// Code generated by csi-proxy-api-gen. DO NOT EDIT.
```

Comment:
Files are getting removed here while in previous PRs it was only about adding changes, is this intended or should this happen later?

Reply:
Yes that's intended. See the note
> Removed the tests for the v1 API since it doesn't follow the versioned API like other API groups, which makes having two sets of tests a lot more work. We're removing v1 soon anyways so this is probably fine.
This means that none of the pkg code uses the v1 client anymore, so it causes the vendor dependency to be deleted.

Reply:
ack